src/modules/cpu/kernel/down_mixing.hpp

```diff
@@ -0,0 +1,49 @@
+#include "rppdefs.h"
+
+RppStatus down_mixing_host_tensor(Rpp32f *srcPtr,
+                                  RpptDescPtr srcDescPtr,
+                                  Rpp32f *dstPtr,
+                                  Rpp64s *samplesPerChannelTensor,
+                                  Rpp32s *channelsTensor,
+                                  bool  normalizeWeights)
+{
+    for(int batchCount = 0; batchCount < srcDescPtr->n; batchCount++)
+    {
+        Rpp32f *srcPtrTemp = srcPtr + batchCount * srcDescPtr->strides.nStride;
+        Rpp32f *dstPtrTemp = dstPtr + batchCount * srcDescPtr->strides.nStride;
+
+        int channels = channelsTensor[batchCount];
+        int64_t samples = samplesPerChannelTensor[batchCount];
+        std::vector<float> weights;
+        weights.resize(channels, 1.f / channels);
+        std::vector<float> normalizedWeights;
+        
+        if (normalizeWeights) 
+        {
+            normalizedWeights.resize(channels);
+
+            // Compute sum of the weights
+            double sum = 0.0;
+            for (int i = 0; i < channels; i++)
+                sum += weights[i];
+            
+            // Normalize the weights 
+            for (int i = 0; i < channels; i++) 
```

Comment:
How long is "channels"? Is this loop parallelizable?

Reply:
Channels will be in lesser range and will be maximum 2 digit number